src/test/java/org/reaktivity/specification/amqp/streams/ConnectionIT.java

```diff
@@ -39,13 +39,37 @@
 
     @Test
     @Specification({
-        "${scripts}/header.exchange/handshake.client",
-        "${scripts}/header.exchange/handshake.server"})
+        "${scripts}/connection/header.exchange/handshake.client",
+        "${scripts}/connection/header.exchange/handshake.server"})
     @ScriptProperty("serverTransport \"nukleus://streams/amqp#0\"")
     public void shouldExchangeHeader() throws Exception
     {
         k3po.start();
         k3po.notifyBarrier("ROUTED_SERVER");
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+        "${scripts}/connection/open.exchange/client",
+        "${scripts}/connection/open.exchange/server"})
+    @ScriptProperty("serverTransport \"nukleus://streams/amqp#0\"")
+    public void shouldExchangeOpen() throws Exception
+    {
+        k3po.start();
+        k3po.notifyBarrier("ROUTED_SERVER");
+        k3po.finish();
+    }
+
+    @Test
+    @Specification({
+        "${scripts}/session/begin.exchange/client",
+        "${scripts}/session/begin.exchange/server"})
+    @ScriptProperty("serverTransport \"nukleus://streams/amqp#0\"")
+    public void shouldExchangeBegin() throws Exception
+    {
+        k3po.start();
+        k3po.notifyBarrier("ROUTED_SERVER");
+        k3po.finish();
+    }
```

Comment:
Please move this and other session scenarios from `ConnectionIT` to `SessionIT`.

Reply:
Created a new class `SessionIT` and moved `shouldExchangeBegin` method to `SessionIT`.